remix.init/index.js

```diff
@@ -238,6 +238,7 @@ const main = async ({ isTypeScript, packageManager, rootDirectory }) => {
     fs.rm(path.join(rootDirectory, ".github", "workflows", "no-response.yml")),
     fs.rm(path.join(rootDirectory, ".github", "dependabot.yml")),
     fs.rm(path.join(rootDirectory, ".github", "PULL_REQUEST_TEMPLATE.md")),
+    fs.rm(path.join(rootDirectory, "LICENSE.md")),
```

Comment:
Does this not go against this portion of the license?

> The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.

Reply:
I guess that applies more to if you clone the blues stack, not install from it?

Reply:
> I guess that applies more to if you clone the blues stack, not install from it?

That's indeed my thought as well